src/sage/matrix/matrix_integer_dense.pyx

```diff
@@ -3229,15 +3235,9 @@ cdef class Matrix_integer_dense(Matrix_dense):
 
             if algorithm == "NTL:LLL":
                 if transformation:
-                    r, det2, UNTL = A.LLL(a,b, verbose=verb, return_U=True)
+                    r, _, UNTL = A.LLL(a,b, verbose=verb, return_U=True)
```

Comment:
Since you touch this line, you could fix the spacing `LLL(a,b, verbose...` to `LLL(a, b, verbose...)`